torchtext/experimental/datasets/raw/text_classification.py

```diff
@@ -35,24 +35,15 @@ class RawTextIterableDataset(torch.utils.data.IterableDataset):
     """Defines an abstraction for raw text iterable datasets.
     """
 
-    def __init__(self, iterator):
+    def __init__(self, iterator, start=0, num_lines=None):
```

Comment:
I understand the allure of "setup_iter" to be in the multiprocessing context. When passed into DataLoader we will want to use the worker info to forward the underlying iterator and prevent batches with repeated datapoints. However, we don't make use of workerinfo in ```__iter__``` right now. Further, specifying an offset and length could be useful for the constructor in general. This could arguably be skipped and a generic abstraction that returns an iterator at a given offset and length could be introduced (albeit it seems unnecessary given how little code it is).

Reply:
I think that's fine to remove those argus since most datasets in torchtext fit the memory.